packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -295,6 +296,13 @@ export function toInstrumentedOdspTokenFetcher(
                     throwOdspNetworkError("tokenIsNull", fetchTokenErrorCode, undefined, undefined, { method: name });
                 }
                 return token;
+            }, (error) => {
+                const tokenError = wrapErrorAndLog(
```

Comment:
I do not think we should log anything here. Error reported by higher level event is sufficient here.